src/Bridges/SymfonyConsole/CreateCommand.php

```diff
@@ -11,53 +11,104 @@
 
 use Nette\Utils\Strings;
 use Nextras;
+use Nextras\Migrations\Entities\Group;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 
 class CreateCommand extends BaseCommand
 {
+	/** content source options */
+	const CONTENT_SOURCE_DIFF = 'diff';
+	const CONTENT_SOURCE_STDIN = 'stdin';
+	const CONTENT_SOURCE_EMPTY = 'empty';
+
+	/** @var string */
+	protected $defaultContentSource = self::CONTENT_SOURCE_DIFF;
+
+
+	/**
+	 * @param  string $defaultContentSource
+	 * @return void
+	 */
+	public function setDefaultContentSource($defaultContentSource)
+	{
+		$this->defaultContentSource = $defaultContentSource;
+	}
+
+
+	/**
+	 * @return void
+	 */
 	protected function configure()
 	{
 		$this->setName('migrations:create');
 		$this->setDescription('Creates new migration file with proper name (e.g. 2015-03-14-130836-label.sql)');
 		$this->setHelp('Prints path of the created file to standard output.');
-		$this->addArgument('type', InputArgument::REQUIRED, 's(tructures), b(asic-data) or d(ummy-data');
+
+		$this->addArgument('type', InputArgument::REQUIRED, $this->getTypeArgDescription());
 		$this->addArgument('label', InputArgument::REQUIRED, 'short description');
+
+		$this->addOption('empty', NULL, InputOption::VALUE_NONE, 'create empty file');
+		$this->addOption('diff', NULL, InputOption::VALUE_NONE, 'use schema diff as file content');
+		$this->addOption('stdin', NULL, InputOption::VALUE_NONE, 'use stdin as file content');
```

Comment:
tohle nechápu, jako že bych měl hodil chybu, kdyby to někdo blbě použil (např. `--empty --diff`)? nebo tam mám někde fakt chybu?